utils/oscap.c

```diff
@@ -181,7 +181,7 @@ static int print_versions(const struct oscap_action *action)
 	printf("\n");
 
 	printf("==== Supported OVAL objects and associated OpenSCAP probes ====\n");
```

Comment:
I think adding a column description, similar to a table header, like:
```
Oval Schema           Oval object            OpenSCAP Probe
```
would add value for newcomers.